sunpy/instr/fermi.py

```diff
@@ -0,0 +1,294 @@
+from __future__ import absolute_import
+from __future__ import division
+
+import numpy as np
+import copy
+import math
+import urlparse
+import urllib, urllib2
+import tempfile
+import datetime
+import os
+import matplotlib.pyplot as plt
+from bs4 import BeautifulSoup
+from sunpy.time import parse_time, TimeRange
+from sunpy import sun
+from sunpy.io.fits import fits
+from astropy.utils.compat.odict import OrderedDict
+
+import astropy.units as u
+from astropy.coordinates import Angle, Longitude, Latitude
+
+def download_weekly_pointing_file(date):
+    date = parse_time(date)
```

Comment:
This should not be before the docstring, and you need a full numpydoc docstring.